MagicBytesValidator/Formats/Bin.cs

```diff
@@ -2,7 +2,9 @@
 
 namespace MagicBytesValidator.Formats;
 
-public class Bin : FileType
+// TODO: Check if correct
```

Comment:
seems to be correct. still need the reminder?
https://datatracker.ietf.org/doc/html/rfc2046#section-4.5.1